ui/app/controllers/jobs/run/templates/index.js

```diff
@@ -1,17 +1,30 @@
 import Controller from '@ember/controller';
 import { action } from '@ember/object';
 import { tracked } from '@glimmer/tracking';
+import { capitalize } from '@ember/string';
 
 export default class JobsRunTemplatesController extends Controller {
   @tracked selectedTemplate = null;
 
   get templates() {
     return this.model.map((templateVariable) => {
       // THIS LOGIC SHOULD LIKELY MOVE TO THE SERIALIZATION LAYER
-      const description = templateVariable.items.description;
```

Comment:
Moving from .items to .keyValues here because the default templates are not Variables, but just Plain Old Javascript Objects.

Reply:
This would be resolved if we just moved the logic into the adapter layer and we could avoid this. It could lead to a bug down the line.